src/HelpDiv/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { PropsWithChildren } from 'react'
+import styled from 'styled-components'
+
+export interface HelpDivProps {
+  background?: string
+  padding?: string
+  margin?: string
+  width?: string
+  height?: string
+  borderRadius?: string
+  leftOrRight?: string
```

Comment:
being picky. Maybe go with 

```
position: 'bottomLeft' | 'bottomRight'
```

Reply:
went with `tailPosition` since `position` is already a css property